utils/sidebarFilter.ts

```diff
@@ -74,9 +77,8 @@ export default function filter(
             itemFilteredByKey[key] = greaterThan(criteria, itemCount);
           }
         } else if (operatorType === FILTER_CONFIG_EXACT) {
-          itemFilteredByKey[key] = item[key]
-            .toLowerCase()
-            .includes(criteria.toLowerCase());
+          itemFilteredByKey[key] =
+            item[key].toLowerCase() === criteria.toLowerCase();
```

Comment:
This will not be good for keyword filtering (for key === 'name'), should match partial matches in that case.